src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -327,10 +330,17 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals
                 name=base_image_trigger.name if base_image_trigger else "defaultBaseimageTriggerName"
             )
 
+    platform_os = None
+    platform_arch = None
+    if os_type or platform:
```

Comment:
Do you need to check here? Can `get_validate_platform` handle the `None` case and provide the default value?

Reply:
yes it can